pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -274,6 +275,20 @@ type BindingSpec struct {
 	// Names of subordinate objects to create
 	SecretName string `json:"secretName"`
 
+	// TODO: replace with PodPresetTemplateSpec
+
+	// The name of a PodPreset to create.  Must be specified with
+	// AlphaPodPresentSpec.  Alpha features may contain bugs and will not be
+	// forward-migratable.
+	// Immutable.
+	AlphaPodPresetName *string `json:"alphaPodPresetName,omitempty"`
```

Comment:
Is name or ObjectMeta better?  Would we ever want to supply labels or annotations as well?